public/js/dist/notes.dev.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+var tableBody = document.getElementById('tableBody');
+
+function update() {
+  var str = "";
+
+  if (localStorage.getItem('itemsJson') != null) {
+    itemsJson = JSON.parse(localStorage.getItem('itemsJson'));
+    itemsJson.forEach(function (element, index) {
+      str += "\n    <tr>\n    <th scope=\"row\">".concat(index + 1, "</th>\n    <td>").concat(element[0], "</td>\n    <td>").concat(element[1], "</td>\n    <td><button id=\"dBtn\" onclick=\"deleted(").concat(index, ");\">Delete</button>  <button id=\"eBtn\">Edit</button></td>\n    </tr>");
+    });
+  }
+
+  tableBody.innerHTML = str;
+}
+
+update();
+
+function deleted(itemIndex) {
```

Comment:
Function name not clear what function do.

Reply:
"delete" would be good function name here. Avoid using past and future tens words.